remoteappmanager/application.py

```diff
@@ -205,27 +205,13 @@ def _reverse_proxy_init(self):
 
     def _db_init(self):
         """Initializes the database connection."""
-        self.db = orm.Database(self.file_config.db_url)
+        self.db = orm.AccountingInterface(self.file_config.db_url)
```

Comment:
I wouldn't go with Interface. Interface has a clear connotation of a fully abstract class. If it's a facade, gateway, manager, call it so. _namingthingsishard_ maybe AccountingDb? AccountingManager? Accountant?